frontend/src/pages/Documents/DocumentUpload.js

```diff
@@ -28,63 +35,185 @@ const styles = {
   }
 };
 
-export default class DocumentUpload extends Component {
-  render = () => {
-    const { storeWorkflowDocument, workflowDocuments } = this.props;
-    const body = (
-      <TableBody>
-        {workflowDocuments.length > 0 ? (
-          workflowDocuments.map((document, index) => (
-            <TableRow key={`${index}-${document.fileName}`}>
-              <TableCell
-                style={{ textAlign: "center", backgroundColor: "#f3f3f3" }}
-                data-test="workflowitemDocumentFileName"
-              >
-                <Typography variant="body1">{document.fileName}</Typography>
-              </TableCell>
-            </TableRow>
-          ))
-        ) : (
-          <TableRow>
-            <TableCell style={{ backgroundColor: "#f3f3f3" }}>
-              <DocumentEmptyState />
+const expression =
+  /https?:\/\/(www\.)?[-a-zA-Z0-9@:%._\+~#=]{1,256}\.[a-zA-Z0-9()]{1,6}\b([-a-zA-Z0-9()@:%_\+.~#?&//=]*)/gi;
+const regex = new RegExp(expression);
```

Comment:
maybe give the variable `expression` another name. is there a way we can check the url in a more readable way (like in api with Joi.url)?